### ISSUE TITLE
music won't play

### ISSUE THREAD

**Issue by mhkgalvez** [Atraci/Atraci#269]
Hi all,

I just downloaded Atraci 0.7.0 for the first time. Unfortunately, when I click any sound it does not play, instead, I get my screen darkened and a red error message on the bottom right corner of the screen: "Oops! Something went wrong." I run Ubuntu 14.04 64 bits. How should I proceed?

**Comment by fa7ad** [comment]
yeah atraci is near-dead atm. we will try and get it back to work, in the mean time... try _mps-youtube_ and _jukesy.com_

**Comment by renanborgez** [comment]
Me too, i can't play anything
Windows 7 x64 SP1

**Comment by mimi89999** [comment]
Youtube stopped support for api V2 :-(

**Comment by Zorgatone** [comment]
Oh, that sucks :(

**Comment by oxodao** [comment]
Same here :(
Why not moving on to torrent like PopcornTime ?

**Comment by ElGatoSaez** [comment]
LimeWire can download just one song from a Torrent, ¿why don't do that, just download full discs? 

Be a force.
discographics will fall.

**Comment by fa7ad** [comment]
Youtube is LEGAL to a certain extent. Torrenting is piracy and thus illegal

**Comment by oxodao** [comment]
Yeah but who cares ? I mean if you are using a tool like PopcornTime, you don't care about piracy... 
Indeed that's a bad thing but most user rather have facility than legality
